src/ontology/mondo-ingest.Makefile

```diff
@@ -598,6 +598,22 @@ update-jinja-sparql-queries:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_replace_annotation_based_mappings.py
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_mapping_annotations_violation.py
 
+
+#################
+##### Ad hoc ####
+#################
+tmp/ORDO_en_4.4.owl:
+	wget https://www.orphadata.com/data/ontologies/ordo/last_version/ORDO_en_4.4.owl -O $@
+
+tmp/ordo-subsets.tsv: tmp/ORDO_en_4.4.owl
+	$(ROBOT) query -i $< --query ../sparql/select-ordo-subsets.sparql $@
+
+reports/ordo-subsets.robot.template.tsv: tmp/ordo-subsets.tsv tmp/mondo.sssom.tsv
```

Comment:
## Adding this file to `mondo` repo
@matentzn @twhetzel Sorry, I don't have time for this tonight and will be on vacation for the next few days. I might be able to do this tomorrow, but if not, @twhetzel can you do it?

- [ ] Add file to that repo? @twhetzel (maybe @joeflack4 if time)

Reply:
@joeflack4 what needs to be done here?

Reply:
So, I didn't see these instructions in the issue, but if this is anything like other stuff I've worked on, I imagine that this robot template will need to somehow get from `mondo-ingest` into `mondo`.

Hmm... there is a difference based on this and what I've worked on previously like this, though. Previously, I've done something such as put the file in the releases of a repo like `medgen`, and then I wrote a goal in `mondo` to `wget` it, and put it in some directory, and then I ran that and committed the file, and that was the whole PR--just the goal and the committed file.

But, since this is coming from `mondo-ingest`, and there's some sort of way that we normally get files from `mondo-ingest`'s codebase directly into `mondo`, maybe there's a different way to do this?
One thing we could do is, rather than putting it in a `mondo-ingest` release, we could merge this into `main`, and then the 'raw' file could be `wget`'ed directly; I hope I'm explaining this well enough. But that's an other option.

Reply:
I am doing that part - dont worry!

Reply:
all we need to do for this is to create the ROBOT file